doc/api/url.md

```diff
@@ -1074,9 +1074,8 @@ console.log(urlToHttpOptions(myUrl));
 ```
 
 ## Legacy URL API
-<!-- YAML
-deprecated: v11.0.0
--->
+
+> Stability: 3 - Legacy: Use the WHATWG URL API instead.
```

Comment:
I think the history is still important, it is useful to know since when Node.js recommends against using this API.

```suggestion
<!-- YAML
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/37784
    description: Deprecation revoked. Status changed to "Legacy".
  - version: v11.0.0
    pr-url: https://github.com/nodejs/node/pull/22715
    description: This API is deprecated.
-->

> Stability: 3 - Legacy: Use the WHATWG URL API instead.
```